website/docs/r/cloudfront_source.html.markdown

```diff
@@ -58,10 +58,11 @@ In addition to the common properties, the following arguments are supported:
      + `type` - (Required) type of polling source. This has to be `S3BucketPathExpression` for `CloudFront` source.
      + `bucket_name` - (Required) The name of the bucket. This is needed if using type `S3BucketPathExpression`. 
      + `path_expression` - (Required) The path to the data. This is needed if using type `S3BucketPathExpression`.
-     + `sns_topic_or_subscription_arn` - (Optional) This is a computed field for SNS topic/subscription ARN. After creating the source via terraform, you can setup [SNS subscription][3] on Sumologic UI and run the terraform apply with the `-refresh-only` flag to update the state with the actual state of the infrastructure. If SNS has been successfully configured and has received a subscription confirmation request `isSuccess` will be true.
+     + `sns_topic_or_subscription_arn` - (Optional) This is a computed field for SNS topic/subscription ARN.
```

Comment:
We should explicitly call this one as "computed". Add a link to the new section you created?

(please fix other places too)

Reply:
Added the link in the lower section along with Common Source Properties.